src/wazuh_modules/inventory_harvester/src/systemInventory/elements/hwElement.hpp

```diff
@@ -73,8 +73,8 @@ class HwElement final
         // Ex: Any value greater than 0
         element.data.host.memory.total = data->totalMem();
 
-        // Ex: Any value greater than 0
-        element.data.host.memory.used = data->usedMem();
+        // Ex: Any value greater than 0. Calculated as total - free
+        element.data.host.memory.used = data->totalMem() - data->freeMem();
```

Comment:
Could we add a check here to not send a value less than zero? Sometimes the information from the agent can't be trusted.

Reply:
Done!